docs/introduction.rst

```diff
@@ -0,0 +1,276 @@
+An application directory is generated for you with all the necessary files for you to configure and prepare for an
+experiment. When configuring an application, you specify the different roles and what types of inputs your
+application uses. In addition, you write the functionality of your application using the NetQASM library.
+
+After creating and configuring an application, you can create an experiment for it using the ``qne experiment create``
+command. Also here an experiment directory is generated with all necessary files. When configuring your experiment
+you can give values to the inputs that were specified when creating your application. You also choose which channels
+and nodes you use in your network and which role is linked to which node. A network consists of channels and each
+channel consists of two nodes. The nodes can communicate with each other using the channel between them.
+
+Once your experiment is configured you are ready to run it using the ``qne experiment run`` command. Your experiment
+is parsed and sent to the NetSquid simulator. After some time your experiment run will be finished and a results
+directory will be generated in which all the results of your experiment are stored.
+
+
+Prerequisites
+-------------
+* A modern Linux or MacOS (10 or 11) 64-bit (x86_64) operating system. If you don't have Linux or MacOS you could run
+  it via virtualization, e.g. using VirtualBox. If you have Windows 10 or 11 you can also use
+  the `Bash on Ubuntu <https://docs.microsoft.com/en-us/windows/wsl/>`_ subsystem.
+* A `virtual environment <https://docs.python.org/3/library/venv.html>`_ should be created and activated before
+  creating an application.
+* Python version 3.7 or higher and pip version 19 or higher.
+* NetQASM makes use of SquidASM for which you need credentials in order to use it. These credentials can be obtained
+  by registering on the forum of `NetSquid <https://forum.netsquid.org/>`_.
+
+Installation
+------------
+To install all the required packages, execute the following command:
+
+.. code-block:: console
+
+    pip install qne-adk
+
+After installing the qne-adk, you can install SquidASM. Replace '{netsquid-user-name}' and '{netsquid-password}' with
+the credentials you registered on `NetSquid <https://forum.netsquid.org/>`_:
+
+.. code-block:: console
+
+    pip install squidasm --extra-index-url=https://{netsquid-user-name}:{netsquid-password}@pypi.netsquid.org
+
+Now everything should be setup and ready in order to create your own applications and experiments and run them on
+the simulator!
+
+Installation from source
+------------------------
+
+The source for the QNE-ADK can also be found at Github. For the default installation execute:
+
+.. code-block:: console
+
+    git clone https://github.com/QuTech-Delft/qne-adk
+    cd qne-adk
+    pip install .
+
+If you want to install for development, install with:
```

Comment:
Maybe we should be more explicit somehow that this is development of ADK instead of application development.